src/bugle_forms/views/home.clj

```diff
@@ -0,0 +1,10 @@
+(ns bugle-forms.views.home)
+
+(def home-content
```

Comment:
suggestion (non-blocking): The `home` in the var name is redundant considering the namespace you've defined it it. It's cleaner to name functions to omit redundancy when calling with the namespace i.e `bugle-forms.views.home/content`